src/utils/event-manager/wheel-input.js

```diff
@@ -64,6 +64,7 @@ export default class WheelInput {
     if (!this.options.enable) {
       return;
     }
+    event.preventDefault();
```

Comment:
Seems that way. The `setTimeout` code originates from v1 and I don't really know its purpose.